deployment/src/test/java/science/aist/sushiya/deployment/it/ExampleRestTest.java

```diff
@@ -0,0 +1,66 @@
+package science.aist.sushiya.deployment.it;
+
+import at.fh.hagenberg.aist.seshat.Logger;
+import org.springframework.http.*;
+import org.springframework.http.converter.StringHttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.lang.reflect.Method;
+
+@ContextConfiguration(locations = {"classpath*:controllerConfigTest.xml"})
+public class ExampleRestTest extends AbstractTestNGSpringContextTests {
```

Comment:
If not already done in the future pull requests, please create yourself an issue to remove the sample code classes in the future